doc/rados/operations/stretch-mode.rst

```diff
@@ -157,28 +157,27 @@ possible, if needed).
 
 .. _Changing Monitor elections: ../change-mon-elections
 
-Stretch Mode Limitations
-========================
-As implied by the setup, stretch mode only handles 2 sites with OSDs.
+Limitations of Stretch Mode 
+===========================
+Stretch mode can be used only with two sites using OSDs.
 
-While it is not enforced, you should run 2 monitors in each site plus
-a tiebreaker, for a total of 5. This is because OSDs can only connect
-to monitors in their own site when in stretch mode.
+Two monitors plus a tiebreaker monitor should be run in each data center, for a
+total of five monitors. While in stretch mode, OSDs can connect only to monitors
+in the data center of which they themselves are a part.
 
-You cannot use erasure coded pools with stretch mode. If you try, it will
-refuse, and it will not allow you to create EC pools once in stretch mode.
+Erasure-coded pools cannot be used with stretch mode. Attempts to use erasure coded pools with stretch mode will fail. Erasure coded pools cannot be created while in stretch mode. 
 
-You must create your own CRUSH rule which provides 2 copies in each site, and
-you must use 4 total copies with 2 in each site. If you have existing pools
-with non-default size/min_size, Ceph will object when you attempt to
-enable stretch mode.
+To use stretch mode, you will need to create a CRUSH rule that provides two
+copies in each data center. Ensure that there are four total copies: two in
```

Comment:
s/copies/replicas/, even though Bladerunner was overrated.

Reply:
Append "An example of such a CRUSH rule is given above".

Reply:
Agreed, and done.

Reply:
I don't agree that Blade Runner is overrated. I do agree that it is very slow.